modules/core/migrations/20240602202521-initial.sql

```diff
@@ -3,12 +3,12 @@ BEGIN;
 
 CREATE TABLE uploads
 (
-    id         SERIAL PRIMARY KEY,
-    name       VARCHAR(255) NOT NULL,
-    url        VARCHAR(255) NOT NULL,
---     uploader_id INT          REFERENCES users (id) ON DELETE SET NULL,
-    mimetype   VARCHAR(255) NOT NULL,
-    size       INT          NOT NULL,
+    id         VARCHAR(255) PRIMARY KEY,
```

Comment:
Why not uuid?

Reply:
> Why not uuid?

ID is actually a file hash to avoid duplicate files, now I think it was a bad idea because of indexing

Reply:
If you're concerned about storage efficiency, then don't. That's a user problem))

Reply:
Changed to int